test/test_dotclear_importer.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "helper"
+require "tempfile"
+
+Importers::Dotclear.require_deps
+
+# Monkey-patch `Jekyll.logger.abort_with()` to not abort tests.
+module Jekyll
+  class LogAdapter
+    def abort_with(topic, message = nil, &block)
+      error(topic, message, &block)
+    end
+  end
+end
+
+class TestDotclearImporter < Test::Unit::TestCase
+  def described_class
+    Importers::Dotclear
+  end
+
+  context "Invalid export file" do
+    should "log graceful error message" do
+      Tempfile.open("bad-export.txt") do |file|
+        file << "Lorem ipsum dolor sit"
+        file.rewind
+
+        output = capture_output { described_class.validate("datafile" => file.path) }
+        error_msg = "#{file.path.inspect} is not a valid Dotclear export file!"
+
+        assert_includes output, error_msg
+      end 
+    end
+  end
+
+  context "Importing with valid export file" do
+    setup do
+      orig_pwd = Dir.pwd
+      @export_file = File.join(orig_pwd, "test/mocks/dotclear.txt")
```

Comment:
If this variable is not used outside the setup method then we don't need it to be an instance variable.

Reply:
This variable is subsequently used to assert expected log message.